packages/bierzo-wallet/src/routes/balance/components/index.tsx

```diff
@@ -73,103 +35,106 @@ interface GetAddressExtensionProps {
-      color="primary"
-      weight="light"
-      inline
-      link
-      onClick={onRegisterUsername}
-    >
-      personalized address.
-    </Typography>
-  </React.Fragment>
-);
+function GetYourAddressWithExtension({ onRegisterUsername }: GetAddressExtensionProps): JSX.Element {
+  return (
+    <React.Fragment>
+      <Typography variant="subtitle2" weight="semibold" color="primary">
+        username*iov
+      </Typography>
+      <Block marginTop={4} />
+      <Typography variant="subtitle1" weight="semibold" gutterBottom>
+        Choose your address
+      </Typography>
+      <Typography variant="body2" color="textPrimary">
+        With IOV you can choose your easy to read human readable address. No more complicated cryptography
+        when sending to friends.
+      </Typography>
+      <Block marginTop={3} />
+      <Typography
+        id={REGISTER_PERSONALIZED_ADDRESS_ROUTE}
+        variant="subtitle1"
+        color="primary"
+        weight="semibold"
+        inline
+        link
+        onClick={onRegisterUsername}
+      >
+        Choose Now
+      </Typography>
+    </React.Fragment>
+  );
+}
 
-const GetYourAddressWithLedger = (): JSX.Element => (
-  <React.Fragment>
-    <Typography variant="h5" align="center" weight="light">
-      You can not register
-    </Typography>
-    <Typography
-      id={REGISTER_PERSONALIZED_ADDRESS_ROUTE}
-      variant="h5"
-      align="center"
-      color="primary"
-      weight="light"
-    >
-      personalized address
-    </Typography>
-    <Block textAlign="center">
-      <Typography variant="h5" weight="light" inline>
-        using{" "}
+function GetYourAddressWithLedger(): JSX.Element {
+  return (
+    <React.Fragment>
+      <Typography variant="subtitle2" weight="semibold" color="primary">
+        username*iov
       </Typography>
-      <Typography variant="h5" weight="semibold" inline>
-        Ledger Nano S
+      <Block marginTop={4} />
+      <Typography variant="body1" weight="light">
+        You can not register
       </Typography>
-    </Block>
-  </React.Fragment>
-);
+      <Typography id={REGISTER_PERSONALIZED_ADDRESS_ROUTE} variant="body1" color="primary" weight="light">
+        personalized address
+      </Typography>
+      <Block textAlign="center">
+        <Typography variant="body1" weight="light" inline>
+          using{" "}
+        </Typography>
+        <Typography variant="body1" weight="semibold" inline>
+          Ledger Nano S
+        </Typography>
+      </Block>
+    </React.Fragment>
+  );
+}
 
-const BalanceLayout = ({
-  iovAddress,
-  balances,
-  onSendPayment,
-  onReceivePayment,
-  onRegisterUsername,
-  rpcEndpointType,
-}: Props): JSX.Element => {
+const BalanceLayout = ({ iovAddress, balances, onRegisterUsername, rpcEndpointType }: Props): JSX.Element => {
   const tickersList = Object.keys(balances).sort();
   const hasTokens = tickersList.length > 0;
+  const walletIcon = <Image src={wallet} alt="wallet ico" />;
```

Comment:
Not sure if `walletIcon` is constantly re-rendering because it's passed as a prop to `PageContent`, but it seems it can be declared outside of the `BalanceLayout` component.